### PR TITLE
More Info profile tab

### PR DESCRIPTION
**Pull request by rohanputhukudy**
### Administrative Info
Monday Board ID: 853642866
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Added Profile screen navigation
- Added More Info tab to the Profile screen

### Testing
How did you confirm your changes worked? 
- Opened the app on an iPhone 12 mini emulator to view the UI
- Checked to make sure web links and tab  navigation worked on the emulator

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

Simulator Screen Shot - iPhone 12 mini - 2021-04-05 at 16 27 09
Simulator Screen Shot - iPhone 12 mini - 2021-04-05 at 16 27 02
Simulator Screen Shot - iPhone 12 mini - 2021-04-05 at 16 27 18

### CODE REVIEW

**Comment by dacruz21** [comment]
Hey thanks! Initial thoughts just poking around the app, I haven't looked at the code yet:

- Email works on my phone just fine!
- The header semicircle is smaller than the one on the home page, and is missing the Words Alive logo
- Saved/Settings/More Info text doesn't look centered within the buttons, they're all hanging a little low
- Same with the big buttons actually
- "Social Media" should be left-aligned (though actually I like the buttons being centered better, let's keep that but left-align the text)
- The contact us form is centering its content vertically on my phone, I think it should start at the top left like most forms.
- The contact us form is also taller than it needs to be, in the mockup it's definitely wider than it is tall.
- The Send button at the bottom is missing a bottom margin.

**Comment by dacruz21** [comment]
Looking good! Last couple of things:

* Contact us box is too narrow but also too tall, can you get it a little closer to the mockup please?
* Contact us form text has changed in the latest mockup, can you update it please? https://xd.adobe.com/view/6426d54d-9267-4e87-9a79-2455abe126d5-3175/screen/0a73310f-c03d-48ae-84c7-d87d0b5dedb3
* Can you i18n everything on this screen? Social Media, Donate, Become a Volunteer, contact us form text, and Send.

File: mobile/src/screens/ProfileScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        <TouchableOpacity onPress={async () => await Linking.openURL("https://www.facebook.com/WordsAliveSD")}>
+          <Image style={[styles.socialPic, styles.notLeftPic]} source={require('../../assets/images/facebook.png')}/>
+        </TouchableOpacity>
+        <TouchableOpacity onPress={async () => await Linking.openURL("https://www.tiktok.com/@wordsalivesd")}>
+          <Image style={[styles.socialPic, styles.notLeftPic]} source={require('../../assets/images/tiktok.png')}/>
+        </TouchableOpacity>
+      </View>
+      <LargeButton text={t("donate")} onPress={async () => await Linking.openURL("https://www.wordsalive.org/donate")}/>
+      <LargeButton text={t("volunteer")} onPress={async () => await Linking.openURL("https://www.wordsalive.org/becomeavolunteer")}/>
+      <Text style={[TextStyles.heading1, {textAlign: 'center', marginTop: 20}]}>
```
Comment (dacruz21) [inline mobile/src/screens/ProfileScreen.tsx:91]: Let's drop this down to h3 now, since the text is so much longer and that's what the mockup shows

**Review by typokign — CHANGES_REQUESTED** [review]
one more thing then lgtm

**Review by typokign — DISMISSED** [review]
LGTM

**Review by allisonbhavsar — CHANGES_REQUESTED** [review]
This is a minor thing, but I think the background color of the profile screen should explicitly be white. The default color looks slightly off if you compare it to the home screen and the mock ups. Also the keyboard still blocks the contact form, but other than that looks great!

**Review by typokign — APPROVED** [review]
I'm getting gray space on the bottom of the Saved tab (probably b/c of the KeyboardAvoidingView) but we're removing that tab anyway for v1 - LGTM, I'll look into it in another PR

**Review by allisonbhavsar — APPROVED** [review]
LGTM

**Pull request merged**